cmd/rook/ceph/sanitize_disk.go

```diff
@@ -0,0 +1,219 @@
+/*
```

Comment:
The code in the `cmd/rook/ceph` package is really intended to be only for command line processing than then call other packages. I'd suggest creating a package `pkg/daemon/ceph/cleanup` with the sanitize code, and even move the dataDirHostPath cleanup code as well.